### ISSUE TITLE
handle.writeLatin1String is not a function on Node 6.5

### ISSUE THREAD

**Issue by leo** [vercel/vercel#23]
Works fine on earlier versions of 6.x.

``` js
net.js:732
      return handle.writeLatin1String(req, data);
                    ^

TypeError: handle.writeLatin1String is not a function
    at createWriteReq (net.js:732:21)
    at Socket._writeGeneric (net.js:702:11)
    at Socket._write (net.js:725:8)
    at doWrite (_stream_writable.js:307:12)
    at clearBuffer (_stream_writable.js:414:7)
    at Socket.Writable.uncork (_stream_writable.js:243:7)
    at ClientRequest._flushOutput (_http_outgoing.js:666:10)
    at ClientRequest.OutgoingMessage._flush (_http_outgoing.js:640:16)
    at _http_client.js:195:10
    at callSocketMethod (_http_client.js:591:7)
```

@quarterto

**Comment by rauchg** [comment]
@leo pretty sure I fixed this issue. @quarterto can you confirm?

**Comment by apaleslimghost** [comment]
Sorry @rauchg, still seeing this issue with now 0.25

**Comment by apaleslimghost** [comment]
@rauchg seems to be fixed with 0.25.1 and above, thanks!

**Comment by leo** [comment]
Yey!
